docs/appendix/changelog.md

```diff
@@ -10,6 +10,242 @@ aliases:
+
+### Added: Power sensor component
+
+You now have the capability to use a power sensor component to monitor the voltage, current, and power consumption within your machine's system.
+
+To integrate your power sensor into Viam, you can [configure a module](/registry/configure/), or select from the following supported models:
+
+- [INA219](/components/power-sensor/ina219/)
+- [INA226](/components/power-sensor/ina226/)
+- [Renogy](/components/power-sensor/renogy/)
+
+### Added: Filter component's data before the cloud
+
+This update allows you to filter data based on specific criteria before syncing it to [Viam's cloud](/services/data/#cloud-sync).
+It equips machines to:
+
+- Remove data that is not of interest
+- Facilitate high-interval captures while saving data based on your defined metrics
+- Prevent the upload of unnecessary data
+
+To learn more, see [this tutorial](/tutorials/configure/pet-photographer/) on creating and configuring a data filtration module.
+
+## August 2023
+
+### Added: Board command added to the CLI
+
+You can now upload manage Linux board definitions more efficiently.
+
+With the `board` command, you can:
+
+- Use the `viam board upload` command to upload a board definition file
+- Specify the board's name, organization, version, and the path to the JSOn file
+
+### Added: Image inspection for ML training
+
+This update enables you to get a closer examination of your image and streamline your image annotation experience by making it easier to add bounding boxes and labels in the **Data** tab.
+
+With the latest improvements, you can now:
+
+- Navigate between images using the arrow keys in the main image view
+- Expand images for a more detailed inspection by clicking the expand button on the right image panel
+- Move between full-screen images effortlessly with the <> arrow buttons or arrow keys
+- Return to the standard view by using the escape key or collapse button
+
+### Added: Duplicate component
+
+You now have the ability to duplicate any config component, service, module, remote, or process.
+This functionality enables the duplication of any configured component and simplifies the process of creating identical configurations for resources.
+
+To use this feature:
+
+- Click on the duplicate component icon at the top right of any resource
+- Optionally, you can modify the component name to distinguish it
+- Adjust any attributes, such as motor pin numbers
+
+## July 2023
+
+### Added: Apple SSO Authentication
+
+Viam now supports sign-up/log-in through Apple Single Sign-On.
+
+Note that currently, accounts from different SSO providers are treated separately, with no account merging functionality.
+
+### Added: Viam VS Code Extension
+
+There is now a Visual Studio Code extension for Viam that offers a set of code snippets, streamlining the process of building robot applications by providing predefined code for common tasks.
```

Comment:
Careful - this isn't Viam-owned, so not appropriate to feature in our change logs
```suggestion
```